src/common/object_info.c

```diff
@@ -0,0 +1,40 @@
+#ifndef OBJECT_H
+#define OBJECT_H
+#include "common.h"
+#include "object_info.h"
+
+#include <stdio.h>
+
+void object_id_print(object_id obj_id) {
```

Comment:
These methods `object_id_print` and `object_id_tostring` should be in `common.c`